test/remote/gateways/remote_simetrik_test.rb

```diff
@@ -13,51 +12,203 @@ def setup
       year: 2022,
       verification_value: '111'
     )
+    @credit_card_invalid = CreditCard.new(
+      first_name: 'Joe',
+      last_name: 'Doe',
+      number: '2440605262758053',
+      month: 7,
+      year: 2022,
+      verification_value: '111'
+    )
     @amount = 100
+    @sub_merchant = {
+      address: 'None',
+      extra_params: {
+      },
+      mcc: '5816',
+      merchant_id: '400000008',
+      name: '885.519.237',
+      phone_number: '3434343',
+      postal_code: 'None',
+      url: 'string'
+    }
+    @psp_info = {
+      id: '0123',
+      name: 'mci',
+      sub_merchant: {
+        id: 'string',
+        name: 'string'
+      }
 
-    setup_options()
+    }
+    @authorize_void_options_success = {
```

Comment:
It seems like `@authorize_void_options_success` and `@authorize_capture_options_success` are the same, and they could simply be one variable called `@authorize_options_success`